CONTRIBUTING.md

```diff
@@ -1,3 +1,43 @@
-# How to become a contributor and submit your own code
+# Contributing Guidelines
 
-For instructions on how to contribute to the Kubernetes project please refer to the official [contributing guide](https://releases.k8s.io/HEAD/CONTRIBUTING.md)
+Read the following guide if you're interested in contributing to cluster-api.
+
+## Contributor License Agreements
+
+We'd love to accept your patches! Before we can take them, we have to jump a couple of legal hurdles.
+
+Please fill out either the individual or corporate Contributor License Agreement (CLA).
```

Comment:
Should we follow the same CLA as top-level K8s?
https://github.com/kubernetes/community/blob/master/CLA.md

Reply:
It seems like the same CLA. I can just link to the main k8s CLA here instead.